code/game/objects/items/weapons/cards_ids.dm

```diff
@@ -517,6 +517,17 @@ var/const/NO_EMAG_ACT = -50
 	access = list(access_legion, access_maint_tunnels, access_external_airlocks, access_security, access_engine, access_engine_equip, access_medical, access_research, access_atmospherics, access_medical_equip)
 	..()
 
+/obj/item/card/id/distress/ap_eridani
+	name = "\improper Eridani identification card"
+	desc = "A high-tech holobadge, identifying the owner as a contractor from one of the many PMCs from the Eridani Corporate Federation."
+	assignment = "EPMC Asset Protection"
+	icon_state = "erisec_card"
+	overlay_state = "erisec_card"
+
+/obj/item/card/id/distress/ap_eridani/New()
+	access = list(access_distress, access_maint_tunnels, access_external_airlocks, access_security, access_medical, access_medical_equip)
```

Comment:
But here you specify a subset of that previous access.

They will most likely end up with the former.
So if they should not be able to access the difference between the two, you need to change something.

Reply:
Rectified.